lib/utils/CSVDatabase.C

```diff
@@ -118,6 +118,24 @@ CSVDatabase::putComplexVector(
     d_fs.close();
 }
 
+void
+CSVDatabase::putStringVector(
+    const std::string& file_name,
+    const std::vector<std::string> data,
+    int nelements)
+{
+    CAROM_ASSERT(!file_name.empty());
```

Comment:
`CAROM_ASSERT` -> `CAROM_VERIFY` will ensure this is checked even in optimized builds.

Reply:
All the usage in CSVDatabase.C and HDFDatabase.C are changed to `CAROM_VERIFY`